src/components/AddDependencySearchResult/AddDependencySearchResult.js

```diff
@@ -127,20 +128,54 @@ class AddDependencySearchResult extends PureComponent<Props> {
 
     const downloadNumColor = getColorForDownloadNumber(hit.downloadsLast30Days);
 
+    // We have to use a custom highlight
+    // to unescape the html
+    const CustomHighlight = connectHighlight(
+      ({ highlight, attribute, hit, highlightProperty }) => {
+        const highlights = highlight({
+          highlightProperty: '_highlightResult',
+          attribute,
+          hit,
+        });
+
+        return highlights.map((part, i) => {
+          // Run the parser on each of the parts
+          const unescaped = (new DOMParser().parseFromString(
+            `<span>${part.value}</span>`,
```

Comment:
Hm, so I did some quick tests in my browser. I created two functions, and tried them both:

```
const v1 = val => new DOMParser().parseFromString(`<span>${val}</span>`, 'text/html').body.textContent

const v2 = val => new DOMParser().parseFromString(val, 'text/html').body.textContent
```

I tried a string with a trailing space, `"hello world "`, and in both cases I get the exact same thing back; it appears to strip out the <span>, which makes sense, since we're calling `.body.textContent` on the end result, which would just be the inner text.

So yeah, I'm confused 🤔does this actually solve the spacing problem?